### ISSUE TITLE
Why fix the learning rate of batch norm to 0?

### ISSUE THREAD

**Issue by soccergame** [miaow1988/ShuffleNet_V2_pytorch_caffe#19]
I don't understand why

**Comment by miaow1988** [comment]
Because there is no parameter to *learn* in the batch norm layer.
In Caffe, the batch normalization consists of a batch norm layer and a scale layer.
Please check the original paper and the implementation of Caffe for more details.

**Comment by soccergame** [comment]
thank you！
